.github/workflows/tests.yml

```diff
@@ -107,7 +107,6 @@ jobs:
           brew tap homebrew/cask-versions
           brew tap homebrew/command-not-found
           brew tap homebrew/formula-analytics
-          brew tap homebrew/linux-dev
```

Comment:
@Bo98 How come?

Reply:
We try get the Debian 7 Dockerfile moved first and then we can archive it before merging this.

Reply:
@Bo98 Cool, fine with me, doesn't need to block this PR. I've merged the linked PR. Would be good to have everything else deleted except Debian 7, move Debian 7 and archive. don't mind if that happens before or after this. Thanks for the work here ❤️